diagnostic.md

```diff
@@ -45,7 +45,7 @@ while (x < 5) {
 
 How many times does the loop run?  What's the value of `x` after the while loop?
 
-_Replace this text with your answer._
+Will run four times. x=8
```

Comment:
The loop will actually run 3 times, not 4 but the value of x is in fact 8.
1st: 1 < 5 so it runs and now x = 2
2nd: 2 < 5 so it runs and now x = 4
3rd: 4 < 5 so it runs and now x = 8
since x is not less than 5, it will not run a 4th time.